src/components/Login/helpers/TokenService.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable require-jsdoc */
```

Comment:
@tatoMa This file seems to be a "common" or "utility". Is it correct that the file is placed in components folder？

Reply:
Yes, the tokenservice may be required by other components. Moved to the util folder. Thx!